pkg/statuscodes/statuscodes.go

```diff
@@ -26,7 +39,6 @@ const (
 	InternalServerError StatusCode = 800
 	NotImplemented      StatusCode = 801
 	Unavailable         StatusCode = 802
-	UnknownError        StatusCode = 803
```

Comment:
What is the impact on back compat from removing this @deregtd? I made the mistake of doing this in imports and I basically had to go add something back in. Its kind of a pain to hunt down sometimes because underlying packages will fail in weird ways.

If there are other places where this exists then we need to prep those PR's now I think if we want to remove this.

Reply:
i had to remove this.  somehow i was searching badly for this on github and it does actually exist.

Reply:
On imports I removed a method and it was used by user imports. I think got things "building in rms" but it would fail on linting. It was super odd and non-obvious to track down. Especially if its in a core library that other people are using. Anyway. We really should open PR's in those other repos to remove them then to avoid people going through the pain I talked about.